# Downhill — a quantitative teardown 🔬
### The duration ladder (return up, Sharpe down) · the isolated term premium · the 2022 drawdown

![Signal: Real](https://img.shields.io/badge/Signal-Real-2ea44f?style=flat-square)
![Tradability: Fragile](https://img.shields.io/badge/Tradability-Fragile-dab617?style=flat-square)
![Is the duration risk worth it?: Not supported](https://img.shields.io/badge/Is_the_duration_risk_worth_it%3F-Not_supported-8b949e?style=flat-square)

The deep companion to the [notebook for the curious](01_for_the_curious.ipynb). The term premium is real but poorly compensated.

> ⚠️ **Not investment advice.** IEF/SHY/BIL, monthly total return (Yahoo), 2003–2026. Sources in [`docs/references.md`](../docs/references.md).

In [1]:
import sys, os
sys.path.insert(0, os.path.abspath("../../.."))  # repo root (quantlab/)
sys.path.insert(0, os.path.abspath(".."))        # study package (downhill/)
%matplotlib inline
import matplotlib.pyplot as plt
plt.rcParams["figure.figsize"] = (10, 5.5); plt.rcParams["axes.grid"] = True
import numpy as np, pandas as pd
from downhill import data, strategy as st
ret = data.fetch_panel()                       # cache-first
al = ret[["IEF","SHY","BIL"]].dropna()
tp = st.term_premium(ret)


## Verdict, up front

| Axis | Stamp | Why |
|---|---|---|
| Signal | **Real** | term premium IEF−BIL +2.2%/yr |
| Tradability | **Fragile** | excess Sharpe 0.32, −23% in 2022 |
| Worth the risk? | **Not supported** | Sharpe falls with duration (1.82→0.52) |

> 💡 *In plain words:* paid for duration, nowhere near enough.

## 1 · The claim, steelmanned

- **H₁:** longer Treasuries earn a positive excess over cash (the term premium).
- **H₂:** extending duration improves the Sharpe.
- **H₃:** the duration risk is mild.

## 2 · So what? — what rides on each

H₁ is the premium's existence; H₂/H₃ decide whether reaching for it is wise. A real-but-poorly-paid premium argues for the short end.

## 3 · How we'd know — the protocol

The three-point duration ladder (CAGR/Sharpe/vol/DD) → the isolated term premium (IEF−BIL) with its t-stat and drawdown.

## 4 · The teardown

### 4.1 The duration ladder — return up, Sharpe down

In [2]:
import pandas as pd
display(pd.DataFrame({'IEF (7-10y)':st.leg_summary(al,'IEF'),'SHY (1-3y)':st.leg_summary(al,'SHY'),'BIL (cash)':st.leg_summary(al,'BIL')}).T[['cagr','sharpe','vol_ann','max_drawdown']].round(3))

,cagr,sharpe,vol_ann,max_drawdown
IEF (7-10y),0.033,0.520,0.067,-0.233
SHY (1-3y),0.019,1.127,0.016,-0.055
BIL (cash),0.014,1.821,0.007,-0.004


> 💡 *In plain words:* CAGR rises with duration (**H₁ holds**) but Sharpe falls 1.82 → 0.52 (**H₂ rejected**) and the 7–10y drawdown is −23% (**H₃ rejected**).

### 4.2 The isolated term premium

In [3]:
print({k:round(v,3) for k,v in st.excess_stats(tp).items()})
ld=tp.loc['2022']; print('2022 term-premium return:', f'{((1+ld).prod()-1):+.1%}')

{'mean_ann': 0.022, 'sharpe': 0.323, 'tstat': 1.408, 'hit_rate': 0.524, 'n': 229}
2022 term-premium return: -16.0%


> 💡 *In plain words:* +2.2%/yr at Sharpe 0.32 (t 1.4), with a brutal 2022 — a thin, lumpy carry.

### 4.3 Why duration disappointed

(1) **Convex, lumpy risk** — quiet roll-down then a −17% year; (2) **compressed premia** — QE/falling rates pushed term premia toward zero (Cochrane-Piazzesi 2005); (3) **the short end is efficient** — bills/1–3y captured most of the safe yield at a fraction of the risk.

## 5 · The verdict

H₁ holds, H₂/H₃ rejected → Signal `REAL`, Tradability `FRAGILE`, the duration trade `NOT SUPPORTED`.

## 6 · Could you trade it?

For most cautious investors the short end dominated risk-adjusted. Extend duration with a *rate view*, not for the carry — the premium didn't pay for its risk.

## 7 · Going further

Forks: (a) a duration-vol-targeted version (size to constant risk); (b) the curve carry+roll signal (steepness) as a timing overlay; (c) TLT (20y+) — even more duration, even worse Sharpe. Backlog: [`docs/pwb_strategies_inventory.md`](../../../docs/pwb_strategies_inventory.md).